deploy/decco/templates/decco-apps-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Maybe you could move the crds to the `crds/` directory, which seems to be a best practice in Helm (see https://helm.sh/docs/chart_best_practices/custom_resource_definitions/)